### PR TITLE
Move to webpack bundling

### PR DESCRIPTION
**Pull request by juliushaertl**
Part 1 of #541 

- [x] Setup webpack
- [x] Convert code style
- [x] Move out url and l10n helper
- [x] stylelint
- [x] krankerl for packaging
- [x] drone build frontend code
- [x] Remove unused documentList jquery widget and simplify loading indicator

0f058e3 and 80736c8 are just code style conversion without changing any logic change, the rest was split in small commits, so it is a bit easier to review.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		jest: true
+	},
+	globals: {
+		t: true,
+		n: true,
+		OC: true,
+		OCA: true,
+		'$': true,
+		'_': true,
+		Vue: true,
```
Comment (skjnldsv) [inline .eslintrc.js]: You don't have any vue files?

Comment (juliushaertl) [reply .eslintrc.js]: Not yet, as I wanted to do the migration in smaller chunks 😉

Comment (skjnldsv) [reply .eslintrc.js]: Ah ok, so let's leave this I guess then :)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	"description": "Collabora online integration",
+	"version": "1.0.0",
+	"authors": [
+		{
+			"name": "Julius Härtl",
+			"email": "jus@bitgrid.net",
+			"role": "Developer"
+		}
+	],
+	"license": "agpl",
```
Comment (gary-kim) [inline package.json]:
```suggestion
	"license": "AGPL-3.0-or-later",
```
You can put the full license in `package.json`.

**Pull request merged**